src/Library/Router/RouterService.ts

```diff
@@ -66,6 +66,9 @@ export class RouterService {
     const keys: Array<Key> = [];
     const regex: RegExp    = pathToRegexp(route, keys);
 
+    // Remove route if previously registered.
+    this.routes = this.routes.filter(target => target.route !== route && target.method !== method);
+
```

Comment:
Goed zo!

Reply:
I had this staged for a while. I forgot to push it...